contracts/Whitelist.sol

```diff
@@ -0,0 +1,20 @@
+pragma solidity 0.5.11;
```

Comment:
Not sure we need to have whitelist

Reply:
Sorry could you add your reviews to https://github.com/barryWhiteHat/maci/pull/12 I accidentally pushed to the wrong origin/master 😓 

I wanted a whitelist because I wanted the authorize the maci contract as the central authority to interact with the merkletrees - this is to isolate the logic of handling the events and the logic of updating/inserting leaves into the merkletree and will be easier to maintain in the long run

Reply:
I reviewed both